src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/component/sw-custom-field-detail/index.js

```diff
@@ -55,8 +65,24 @@ export default {
             return [this.$root.$i18n.fallbackLocale.value];
         },
 
+        customFieldNameError() {
+            if (!this.currentCustomField.name) {
+                return null;
+            }
+
+            const isValid = CUSTOM_FIELD_NAME_PATTERN.test(this.currentCustomField.name);
```

Comment:
isn't that checked in the backend? why is it then necessary to do it here again?

Reply:
@jleifeld what do you think about that?